ruy/CMakeLists.txt

```diff
@@ -416,7 +416,7 @@ elseif(CMAKE_SYSTEM_PROCESSOR STREQUAL s390 OR CMAKE_SYSTEM_PROCESSOR STREQUAL s
 elseif(CMAKE_SYSTEM_NAME STREQUAL Fuchsia)
   set(ruy_6_cpuinfo "")
 else()
-  set(ruy_6_cpuinfo "cpuinfo")
+  set(ruy_6_cpuinfo "cpuinfo::cpuinfo")
```

Comment:
This file is generated - as explained in the comment at the top. If it needs to be updated, please to try to achieve that by updating and rerunning the generator script instead.

Please also help me understand why this change is needed - as you added those `add_library(cpuinfo::cpuinfo ALIAS cpuinfo)` in CMakeLists.txt, that sounded like that was taking care of that?

Reply:
`add_library(cpuinfo::cpuinfo ALIAS cpuinfo)` works the other way round, it defines `cpuinfo::cpuinfo` as an alias for `cpuinfo` in the case where cpuinfo is used as subdirectory; when used through `find_package`, `cpuinfo-targets.cmake` defines the `cpuinfo::cpuinfo` target.

This is the idiomatic way to consume CMake packages, see for example https://github.com/abseil/abseil-cpp/blob/1963f10ae5cb32a7ea6d96b928f69d3c7fba0139/CMakeLists.txt#L140